src/components/ContainerSettings.react.js

```diff
@@ -65,6 +65,11 @@ var ContainerSettings = React.createClass({
                   Advanced
                 </li>
               </Router.Link>
+              <Router.Link to="ContainerSettingsStats" params={{name: container.Name}}>
+                <li>
+                  Status
```

Comment:
Rename to `Stats` - Status is something different and shown in the top title bar. 
It's truly `resource usage statistics`